src/dmd/globals.d

```diff
@@ -316,7 +316,8 @@ extern (C++) struct Global
     Array!(const(char)*)* path;         // Array of char*'s which form the import lookup path
     Array!(const(char)*)* filePath;     // Array of char*'s which form the file import lookup path
 
-    string _version;
+    enum string _version = import("VERSION");
+    enum uint _versionNumber = calcVersionNumber(_version);
```

Comment:
Headers.

Reply:
Aren't enums only existent at CT and not passed to the layout?

Reply:
> Aren't enums only existent at CT and not passed to the layout?

Yes. But there is still the field in the headers.

Reply:
Fair enough. I set `_version` to private, removed it from the header and exposed it to C++ via `versionString` and added a small test to `cxx-frontend`. I hope that works for GDC?